src/excludes.rs

```diff
@@ -0,0 +1,70 @@
+// Copyright 2017 Julian Raufelder.
+
+//! Restore from the archive to the filesystem.
+
+use globset::{Glob, GlobSet, GlobSetBuilder};
+
+use super::*;
+
+#[derive(Debug, Default)]
+pub struct Excludes {}
+
+
+impl Excludes {
+    pub fn parse_excludes(&self, excludes: Option<Vec<&str>>) -> Result<Option<GlobSet>> {
+        match excludes {
+            Some(excludes) => {
+                let mut builder = GlobSetBuilder::new();
+                for exclude in excludes {
+                    builder.add(Glob::new(exclude)
+                        .chain_err(|| format!("Failed to parse exclude value: {}", exclude))?);
+                }
+                Ok(Some(builder.build()
+                    .chain_err(|| "Failed to build exclude patterns")?))
+            }
+            None => Ok(None)
```

Comment:
Rather than an `Option<GlobSet>` can we make it return a `GlobSet` that matches nothing? That might make it easier for the callers.